templates/artist/artist_detail.html

```diff
@@ -22,7 +22,7 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <img src="{{ artist.photo.img.url }}" />
+    {% include "artist/includes/artist_detail.html" with artist=artist %}
```

Comment:
Not 100% certain what you mean here. What part of the {% thumbnail %}...{% endthumbnail %} goes in place of templates/....?

Also, does {% load thumbnail %} just go at the top of the page? Any specific order?